container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/kubernetes_label_key/label_key_consts.go

```diff
@@ -28,6 +28,14 @@ const (
 
 	// As of 2022-05-17, these get attached to files artifact expansion volumes
 	userServiceGuidKeyStr = labelKeyPrefixStr + "user-service-guid"
+
+	// We create a duplicate of the enclave uuid and service uuid label key because:
+	// the logs aggregator (vector) needs the enclave uuid and service uuid label keys to create the filepath where logs are stored in persistent volume
+	// but vectors template syntax can't interpret the "kurtosistech.com/" prefix, so we can't use the existing label keys
```

Comment:
Might be worth double-checking - the `kurtosistech.com/` domain prefixing is, to my knowledge, standard Kubernetes practice to ensure you don't have label collisions in your cluster

Reply:
unfortunately vector template syntax will interpret the `.` as nested key. I do think this is a valid concern though but think its okay to wait  to see if users have friction with this or not. as of right now, users usually use kurtosis standalone in a k8s cluster.